### ISSUE TITLE
Button Issue - Stop timer not appearing anymore

### ISSUE THREAD

**Issue by gairehari** [harvesthq/platform#61]
Did anything change in https://platform.harvestapp.com/assets/platform.js recently ?

I have a harvest button in my app and the "Stop Timer" never appears after we start the timer. It was working before.

I was able to replicate the issue with this small html code also.

<!DOCTYPE html>
<html>
    <head>
            <script>
                window._harvestPlatformConfig = {
                    "applicationName": "ExampleCompany",
                    "permalink": "https://example.com/items/%ITEM_ID%"
                };
            </script>
            <script async src="https://platform.harvestapp.com/assets/platform.js"></script>
    </head>
    <body>
        <div class="harvest-timer"
            data-item='{"id":1337,"name":"Write documentation"}'>
        </div>
    </body>
</html>

**Comment by pusewicz** [comment]
@gairehari Thanks for getting in touch! Are you testing this in Safari, by any chance? When _Prevent cross-site tracking_ is enabled, Platform does not work correctly. It needs this option turned off. Same applies to third-party cookies. Hope this helps!

**Comment by gairehari** [comment]
@pusewicz This is in chrome. It started happening recently. It used to work perfectly before.

**Comment by pusewicz** [comment]
Could you double check that _Block third-party cookies_ is not enabled in Chrome?

**Comment by gairehari** [comment]
@pusewicz  It is not enabled.

Did you try the jsfiddle code above ?

**Comment by pusewicz** [comment]
@gairehari Yes. With the setting _Block third-party cookies in Icognito_ it works as expected.

**Comment by gairehari** [comment]
@pusewicz My chrome has "Block third-party cookies" enabled too but I see same issue in both normal and incognito windows. (Same issue in firefox too)

**Comment by pusewicz** [comment]
That's the problem at hand. If you have blocking enabled it will _not_ work. In order for Platform to work cookies should not be blocked in the normal mode, only in Incognito.

**Comment by gairehari** [comment]
@pusewicz  Sorry for the confusion. I have cookies blocked only in incognito and not on normal mode. 

I see issues in both incognito and normal mode (and in other browsers too)

**Comment by kostyakomarov** [comment]
The issue is still relevant. Can you suggest a solution?
